test/unit/spec/common/jobManager-Spec.js

```diff
@@ -1076,41 +1224,56 @@ define([
                     spyOn(this.jobManagerInstance, 'updateModel').and.callThrough();
 
                     // trigger the update
-                    this.jobManagerInstance.runHandler(
-                        jcm.MESSAGE_TYPE.STATUS,
-                        { jobState: batchParentUpdate, [jcm.PARAM.JOB_ID]: jobId },
-                        jobId
-                    );
-                    expect(this.jobManagerInstance.model.getItem(`exec.jobState`)).toEqual(
-                        batchParentUpdate
-                    );
-                    expect(
-                        this.jobManagerInstance.listeners[jobId][jcm.MESSAGE_TYPE.STATUS]
-                    ).toBeDefined();
-                    expect(this.jobManagerInstance.updateModel).toHaveBeenCalledTimes(1);
-                    expect(console.error).toHaveBeenCalledTimes(1);
-
-                    // all child jobs should have job status and job info listeners
-                    batchParentUpdate.child_jobs.forEach((_jobId) => {
-                        expect(
-                            this.jobManagerInstance.listeners[_jobId][jcm.MESSAGE_TYPE.STATUS]
-                        ).toBeDefined();
+                    return new Promise((resolve) => {
+                        this.jobManagerInstance.addListener(
+                            jcm.MESSAGE_TYPE.STATUS,
+                            jcm.CHANNEL.JOB,
+                            jobId,
+                            {
+                                zzz_job_action: () => {
+                                    resolve();
+                                },
+                            }
+                        );
+                        this.bus.send(
+                            { jobState: batchParentUpdate, [jcm.PARAM.JOB_ID]: jobId },
+                            {
+                                channel: { [jcm.CHANNEL.JOB]: jobId },
+                                key: { type: jcm.MESSAGE_TYPE.STATUS },
+                            }
+                        );
+                    }).then(() => {
+                        expect(this.jobManagerInstance.model.getItem(`exec.jobState`)).toEqual(
```

Comment:
no changes to the test content other than the nesting, but prettier has made this look very different

Reply:
Prettier has some strong opinions. Mostly, I think its opinion is, "Make pull requests look 10 times as large as they should."